packages/IMAPClient-Core.package/ICFolder.class/instance/sanitizeMailCollection..st

```diff
@@ -1,4 +1,4 @@
-email access
+validating
```

Comment:
I think this is the wrong category for this method. I my opinion it changes the emails and does not only validate something.